httpretty/setup.py.patch

```diff
@@ -0,0 +1,14 @@
+--- setup.py	2015-05-18 18:10:10.000000000 +0000
++++ setup_modified.py	2016-12-06 18:43:52.209700651 +0000
```

Comment:
That doesn't change the file name (_I think_). It's just metadata saying how the diff was computed.